Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: clean rpm
```

Comment:
Can we please move this out of the top-level directory? Having a _Makefile_ available in the project root is bound to confuse someone.

Reply:
The intention is to make the build process for local development and CI the same over time, so you use the same commands, i.e. "make deb" builds debian artifacts.  Sure we can move things around.